sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,995 @@
+Stacks-on-Bitcoin transaction).
+
+## Freezing and Thawing
+
+In the preceding sections, there are failure modes whereby the system can
+freeze.  This can happen if the Stackers fail to execute a WSTS signing round or
+DKG, or fail to vote for a new sBTC wallet redeem script.  If the system freezes,
+then STX unlocks do not happen, and PoX payouts do not happen either, for as
+long as the system is frozen.  Instead, miners pay to a burn address, and the PoX
+system refuses to unlock STX.  The only way the STX can unlock and PoX payments
+resume is if the system can be thawed.  If the system remains frozen across two
+reward cycle boundaries -- i.e. the current set of Stackers remain frozen and
+the subsequent set remain frozen, then the Stacks blockchain halts.
+
+To thaw the system:
+
+* On deposit, the system can freeze if Stackers cannot complete a WSTS signing
+  round to vote to accept, reject, or abstain from a deposit request, and the
+deposit requests remains pending for too long.  To thaw
+the system, the Stackers would need to come back online and acknowledge all
+pending deposit transactions.  If the system remains frozen across a prepare
+phase, then the system can be thawed by the Stackers passing the BTC to the new
+set of Stackers, which would cancel the pending deposit requests.
+
+* On withdraw, the system can freeze if Stackers cannot complete a timely signing
+  round to vote to process the withdraw.  To thaw the system, Stackers would need
+to come back online to either fulfill all pending withdraw requests, or NACK them.
+If the system remains frozen across a reward cycle boundary, then it
+thaws if the old Stackers are able to transfer their
+BTC to the new Stackers.  This act will cancel all pending sBTC withdraw
+requests.
+
+* On BTC-transfer, the system can freeze in one of two ways:  the new Stackers
+cannot vote on a new aggregate public key and redeem script before the end of
+the reward cycle, or the old Stackers cannot hand off all of the BTC to the new
+Stackers by the start of the new reward cycle. The only way to thaw the system
+is for the new Stackers to eventually complete their vote and the old Stackers
+send over the BTC before the next PoX anchor block is mined.  If the system
+remains frozen this way, and the next PoX anchor block is mined, then the Stacks
+blockchain halts.
+
+### System Recovery from Frozen Wallet
+
+In the event that the BTC gets "stuck" and Stackers are unable to access it, the
+system will support a variation of a deposit transaction called a _donation_.
+The only difference between a deposit and a donation is that the donation does
+not materialize sBTC.  It merely gives Stackers some BTC with which to fulfill
+withdraws.  This feature is meant to enable Stackers to recover lost BTC and
+unfreeze the system, should the need ever arise.
+
+## Capital Limits
+
+**No capital limit will be imposed on the system.** Instead, the protocol will rely
```

Comment:
@andreserrano This will likely stay the same through future iterations, so please make any alterations.